crates/zome_types/src/capability/secret.rs

```diff
@@ -40,9 +40,10 @@ impl<'de> serde::de::Deserialize<'de> for CapSecret {
     {
         let bytes: &[u8] = serde::de::Deserialize::deserialize(deserializer)?;
         if bytes.len() != CAP_SECRET_BYTES {
+            let exp_msg = format!("a cap secret length of: {} bytes; got: {} bytes", CAP_SECRET_BYTES, bytes.len());
```

Comment:
```suggestion
            let exp_msg = format!(
                "expected a cap secret length of: {} bytes; got: {} bytes",
                CAP_SECRET_BYTES,
                bytes.len()
            );
```

should fix fmt failing test